Its.Log.Monitoring/Telemetry.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections.Generic;
+using Its.Log.Instrumentation;
+
+namespace Its.Log.Monitoring
+{
+    public class Telemetry
+    {
```

Comment:
Differentiating this from the `Its.Log.Telemetry` class is probably a good idea. They look awfully similar. What's the difference in implementation and in intention?